quodlibet/ext/events/mpris/mpris2.py

```diff
@@ -279,7 +280,13 @@ def ignore_overflow(dbus_type, value):
                 metadata["xesam:" + xesam] = list(map(unival, vals))
 
         # All single values
-        sing_val = {"album": "album", "title": "title", "asText": "~lyrics"}
+        columns = get_columns()
+        title_tag = "~title~version" if "~title~version" in columns \
+                    else "title"
+        album_tag = "~album~discsubtitle" if "~album~discsubtitle" in columns \
+                    else "album"
```

Comment:
It's a shame perhaps we don't save that setting, as this _feels_ hacky to need to do.


```suggestion
        title_tag = ("~title~version" if "~title~version" in columns 
                     else "title")
        album_tag = ("~album~discsubtitle" if "~album~discsubtitle" in columns
                     else "album")
```

Reply:
@declension ah yes, took me a moment to figure this out :grinning: 

Thanks for merging!